### ISSUE TITLE
[Feature]: DHCP / Zeroconf discovery for Ajax hubs on the local network

### ISSUE THREAD

**Issue by bvis** [bvis/aegis-hass#92]
### Description

The integration currently requires fully manual setup (account + app label + space selection). When the Ajax hub is on the same LAN as Home Assistant — which is the common case — HA can show the integration as a "Discovered" entry in **Settings → Devices & Services** so the user only has to click "Configure" and enter credentials, instead of searching for "Aegis" first.

Two HA discovery hooks apply:

1. **DHCP** — HA inspects DHCP requests and triggers `async_step_dhcp` on integrations whose `manifest.json` declares matching MAC prefixes (OUIs) or hostnames. We need to fill in the `dhcp` array in the manifest.
2. **Zeroconf / mDNS** — Some Ajax hubs advertise mDNS services on the LAN. Worth a quick capture to confirm what they advertise (likely `_ajax._tcp.local.` or vendor-specific) before declaring it.

### Use case

- New users find Aegis without knowing the integration name.
- Existing users with multiple hubs get a fast path to add a second hub.
- Lower setup friction generally → fewer "how do I install this" issues.

### Related Ajax devices

All Ajax hubs reachable on the LAN (Hub family + Hub Hybrid + Hub Fibra + Hub Mega + Hub Superior).

### Additional context

Implementation steps:

1. Capture the MAC OUI(s) used by Ajax hubs we have access to (Hub Hybrid 4G, Hub 2 Plus, Hub Plus). The vendor's IEEE OUI registration is the authoritative source — cross-check with `arp -a` while a hub is on the LAN.
2. Add to `manifest.json`:
   ```json
   "dhcp": [
     {"macaddress": "XX:XX:XX:*"},
     ...
   ]
   ```
3. Implement `async_step_dhcp(self, discovery_info)` in `config_flow.py`:
   - Use the discovered MAC as `unique_id` if we don't already have an entry for the same Ajax account+space.
   - If credentials already exist for an account that owns this hub, abort with `already_configured`.
   - Otherwise forward into the existing user/credentials step with a context flag so the UI says "An Ajax hub was found on your network".
4. Optional: add `async_step_zeroconf` once we confirm what the hub advertises.
5. Tests: synthesise DHCP discovery info in the config-flow test suite, assert the right next-step / abort paths.

Caveats:
- Only helps when the hub is on the same network as HA. Cellular-only or off-LAN hubs are unaffected — that's fine, manual setup still works.
- We won't have credentials at discovery time; we still need the user to log in. The discovery's value is naming the integration unambiguously and pre-filling nothing more than that.

**Comment by bvis** [comment]
Holding this off for now: implementing the discovery flow without real Ajax-hub MAC OUIs would either be dead code (with no `dhcp` entries in `manifest.json` HA never invokes `async_step_dhcp`) or generate false-positive discovery cards if I guess the wrong OUI.

What's needed to land this:

1. Capture the actual OUIs from one or two real hubs on the LAN: while the hub is connected by Ethernet or Wi-Fi, run `arp -a | grep -i <hub-ip>` (or `ip neigh` on Linux) from any other machine on the same network. The first three octets of the MAC are the OUI to add.
2. Cross-check what the hub broadcasts as DHCP `hostname` (`tcpdump -nei <iface> port 67 or port 68` for one renewal cycle).
3. Add the captured OUI(s) and hostname(s) to `dhcp` in `manifest.json` and ship the `async_step_dhcp` implementation.

Once the OUI capture is done I can wrap this up in one short PR. Until then, prioritising #89, #90, #91.
